gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -164,11 +177,22 @@ def run(argv=None):
             'large number of files are specified by input_pattern. '
             'Note that each VCF file must still contain valid header files '
             'even if this is provided.'))
+  parser.add_argument(
+      '--allow_malformed_records',
+      type='bool', default=False, nargs='?', const=True,
+      help=('If true, failed VCF record reads will not raise errors. '
+            'Failed reads will be logged as warnings and returned as '
+            'MalformedVcfRecord objects.'))
+  parser.add_argument(
+      '--optimize_for_large_inputs',
+      type='bool', default=False, nargs='?', const=True,
+      help=('If true, the pipeline runs in optimized way for handling large '
+            'inputs. Set this to true if you are loading more than 50,000 '
+            'files.'))
```

Comment:
Could we remove this and automatically set it for > 50000 files?

Reply:
yeap :) once we have your header changes merged, we'll do this automatically for >50k input sizes. We'll keep this option for users who know that their input size will be large (it takes a while to get the list of files, which would be a waste of time).